src/main/java/xyz/nucleoid/plasmid/command/MapCommand.java

```diff
@@ -193,6 +224,74 @@ private static int addRegion(CommandContext<ServerCommandSource> context) throws
         return Command.SINGLE_SUCCESS;
     }
 
+    private static int renameRegions(CommandContext<ServerCommandSource> context, BlockStateVariantMap.TriFunction<TemplateRegion, String, BlockPos, Boolean> predicate) throws CommandSyntaxException {
```

Comment:
Might make sense to use a custom interface for the predicate- that is one real chunky boi :D